include/misc/wallinTerran.hpp

```diff
@@ -80,8 +80,8 @@ namespace ghost
   vector< Building > makeTerranBuildings()
   {
     // Academy
-    Building a1 = factoryTerranBuilding("Academy");
-    Building a2 = factoryTerranBuilding("Academy");
+//     Building a1 = factoryTerranBuilding("Academy");
```

Comment:
As I said, I think for comparison purposes it's better for a fair comparison... But in any case the building list should be something more dynamic...